src/snapshotter.js

```diff
@@ -125,6 +125,7 @@ export default class SnapShotter {
     try {
       await this.driver.wait(this._until.elementIsVisible(element), timeout);
     } catch (error) {
+      console.log('');
```

Comment:
Do we need this?

Reply:
kinda 
if you stick it in the logger it does a weird 
[]
[]

could probably do with a comment explaining its there to make the log output look nice